pkg/plugins/runtime/k8s/controllers/gateway_instance_controller.go

```diff
@@ -327,9 +327,13 @@ func GatewayToInstanceMapper(l logr.Logger, client kube_client.Client) kube_hand
 	return func(obj kube_client.Object) []kube_reconcile.Request {
 		gateway := obj.(*mesh_k8s.MeshGateway)
 
+		spec, err := gateway.GetSpec()
+		if err != nil {
+			l.WithValues("gateway", obj.GetName()).Error(err, "failed to get core resource from MeshGateway")
```

Comment:
isn't the mesh also useful to identify the gateway?

Reply:
Sure, I just used the existing log message in this function

Reply:
In fact it's impossible to get the mesh because this is the step where we convert the JSON into a MeshGateway